dspace/config/crosswalks/google-metadata.properties

```diff
@@ -73,5 +73,9 @@ google.citation_patent_number =
 google.citation_technical_report_number =
 google.citation_technical_report_institution = dc.publisher
 
+#priority whitelist for citation_pdf_url, shortnames are defined in dspace/config/registries/bitstream-formats.xml
+#priority order is defined here, where the first type is the most important
+google.citation.prioritized_types = Adobe PDF, Microsoft Word, Microsoft Word XML, RTF, Photoshop, EPUB
```

Comment:
I think that the PS format from the original recommended whitelist (PDF, PS, DOC/DOCX, RTF, EPUB) should be PostScript and not Photoshop. This whitelist should list textual formats.